spec/factories/pieces.rb

```diff
@@ -8,31 +8,46 @@
     icon ''
     color 'white'
 
-    factory :pawn, class: Pawn, parent: :piece do
+    factory :pawn, class: Pawn do
```

Comment:
yes it worked for me! celia helped me out with this. didn't seem to give me any issues :)